src/main/java/io/jenkins/plugins/jfr/JavaFlightRecorderAction.java

```diff
@@ -69,6 +72,22 @@ public void doSessions(StaplerRequest req, StaplerResponse rsp) throws IOExcepti
         }
     }
 
+    @RequirePOST
+    @WebMethod(name = "dump")
+    public void doDump(StaplerRequest req, StaplerResponse rsp) throws IOException {
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+        rsp.setContentType("application/json");
+        rsp.setCharacterEncoding("UTF-8");
+        try (Writer writer = rsp.getWriter();
+                Reader reader = req.getReader()) {
+            DumpRequest dumpRequest = objectMapper().readValue(reader, DumpRequest.class);
+            DumpResponse dumpResponse = service.dump(dumpRequest);
+            objectMapper().writeValue(writer, dumpResponse);
```

Comment:
Extract `objectMapper()` to a variable and use the variable twice rather than invoking the method twice.